src/sass/components/_about.scss

```diff
@@ -2,21 +2,47 @@
   max-width: $mobile-width;
   margin: 0 auto;
   padding: 52px 0;
-  border-bottom: 2px solid $color-dark;
+  &__container {
+    @include flex(column, center, center, wrap);
+  }
+
+  @include media-desktop {
+    max-width: $desktop-content-width;
+    @include flex(column, center, center, wrap);
+  }
 
   &__image {
     width: 100%;
+    @include media-desktop {
```

Comment:
as I understand `media-desktop` expands into `@media (min-width: 375px)`. this doesn't sound as `desktop` resolution. I'd probably change the name of this mixin. there are a couple of widely used approaches on how to call your resolutions: `xs, sm, md, lg, xl` or `mobile, tablet, laptop, desktop`. so in this case it sounds more like `sm` or `tablet`. in responsive design it usually means that css will be applied to `sm` resolution and above